### PR TITLE
Sc 15 add validator

### PR DESCRIPTION
**Pull request by akp111**

### CODE REVIEW

File: src/addDelegator.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export async function getUnsignedAddDelegator(
   }
 }
 
-async function getAddDelegatorParams(
+export async function getAddDelegatorParams(
```
Comment (avisingh2) [inline src/addDelegator.ts:62]: please add documentation for all the exported function, interfaces, constants and enums

File: src/addValidator.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ export async function getUnsignedAddValidator(
   }
 }
 
-async function getAddValidatorParams(
+export async function getAddValidatorParams(
```
Comment (avisingh2) [inline src/addValidator.ts:66]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 3]
```diff
@@ -163,7 +163,7 @@ async function contextFromOptions(options: OptionValues): Promise<Context> {
 
 // Network is obtained from context file, if it exists, else from --network flag.
 // This is because ledger does not need a context file
-function networkFromOptions(options: OptionValues): string {
+export function networkFromOptions(options: OptionValues): string {
```
Comment (avisingh2) [inline src/cli.ts:166]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 4]
```diff
@@ -176,7 +176,7 @@ function networkFromOptions(options: OptionValues): string {
   return network
 }
 
-function getOptions(program: Command, options: OptionValues): OptionValues {
+export function getOptions(program: Command, options: OptionValues): OptionValues {
```
Comment (avisingh2) [inline src/cli.ts:179]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 5]
```diff
@@ -189,7 +189,7 @@ function getOptions(program: Command, options: OptionValues): OptionValues {
   return { ...allOptions, network }
 }
 
-function capFeeAt(cap: number, usedFee?: string, specifiedFee?: string): void {
+export function capFeeAt(cap: number, usedFee?: string, specifiedFee?: string): void {
```
Comment (avisingh2) [inline src/cli.ts:192]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 6]
```diff
@@ -298,7 +298,7 @@ export async function initCtxJsonFromOptions(options: OptionValues): Promise<voi
 //////////////////////////////////////////////////////////////////////////////////////////
 // Network info
 
-function logAddressInfo(ctx: Context) {
+export function logAddressInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:301]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 7]
```diff
@@ -307,7 +307,7 @@ function logAddressInfo(ctx: Context) {
   log(`secp256k1 public key: 0x${compressedPubKey}`)
 }
 
-async function logBalanceInfo(ctx: Context) {
+export async function logBalanceInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:310]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 8]
```diff
@@ -317,7 +317,7 @@ async function logBalanceInfo(ctx: Context) {
   log(`P-chain ${ctx.pAddressBech32}: ${pbalance} FLR`)
 }
 
-function logNetworkInfo(ctx: Context) {
+export function logNetworkInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:320]: please add documentation for all the exported function, interfaces, constants and enums

File: src/cli.ts [hunk 9]
```diff
@@ -326,7 +326,7 @@ function logNetworkInfo(ctx: Context) {
   log(`assetId: ${ctx.avaxAssetID}`)
 }
 
-async function logValidatorInfo(ctx: Context) {
+export async function logValidatorInfo(ctx: Context) {
```
Comment (avisingh2) [inline src/cli.ts:329]: please add documentation for all the exported function, interfaces, constants and enums

File: test/unit/cli.test.ts [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+} from '../../src/cli';
+import { logInfo, log } from '../../src/output';
+import { contextEnv } from '../../src/constants';
+import { Context } from '../../src/interfaces';
+import fixtures from '../fixtures/cli.data';
+jest.mock('../../src/output', () => ({
+  logInfo: jest.fn(),
+  log: jest.fn()
+}));
+describe('cli Testcases', () => {
```
Comment (avisingh2) [inline test/unit/cli.test.ts:18]: put the blank line between imports and business log

**Comment by akp111** [comment]
Addressed in #17

**Pull request closed without merging**